server/jetstream_api.go

```diff
@@ -193,7 +193,14 @@ const (
 	// JSApiServerStreamMove is the endpoint to move streams off a server
 	// Only works from system account.
 	// Will return JSON response.
-	JSApiServerStreamMove = "$JS.API.SERVER.STREAM.MOVE"
+	JSApiServerStreamMove  = "$JS.API.ACCOUNT.STREAM.MOVE.*.*"
```

Comment:
Why the choice of removing account and stream from the request object and instead making part of the subject?

Reply:
This expanded from one subject to two and I know that there will be a similar one going forward.
It seemed better to have that align with stream/consumer as well and identify the resource in the subject.